core-plugins/src/main/java/co/cask/hydrator/plugin/transform/LogParserTransform.java

```diff
@@ -102,6 +102,15 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) throws Ille
       LOG.error("Log format not currently supported.");
       throw new IllegalStateException("Unsupported log format: " + config.logFormat);
     }
+    Schema inputSchema = pipelineConfigurer.getStageConfigurer().getInputSchema();
+    if (inputSchema != null) {
+      if (!validateInputSchemaType(inputSchema.getType())) {
+        throw new IllegalArgumentException(
```

Comment:
its also called from transform, where we don't want to throw exception. that's why i return a boolean from that validate method and throw exception here.